service/src/main/java/com/epam/esm/service/ParamContainerService.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.esm.service;
+
+import com.epam.esm.dto.ParamContainer;
+import com.epam.esm.dto.certificate.GiftCertificateResponseDTO;
+import com.epam.esm.entity.GiftCertificate;
+
+import java.util.List;
+
+public interface ParamContainerService {
```

Comment:
What is the purpose of this service? You should use GiftCertificateService and add `findGiftCertificateByIdWithTagsAndParams` method there